test-functional/data/simple-metadata/prompt-removed/v0/package.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <types>
```

Comment:
@froucher Please remove these lines with the wildcard for Promt (from all package.xml files).

Reply:
@froucher there are two more wildcards:

- `test-functional/data/simple-metadata/prompt-updated/v0/package.xml`
- `test-functional/data/simple-metadata/prompt-updated/v1/package.xml`